badger/cmd/write_bench.go

```diff
@@ -112,7 +115,14 @@ func writeRandom(db *badger.DB, num uint64) error {
 	for i := uint64(1); i <= num; i++ {
 		key := make([]byte, keySz)
 		y.Check2(rand.Read(key))
-		if err := batch.Set(key, value); err != nil {
+
+		e := badger.NewEntry(key, value)
+
+		if withTTL {
+			e = badger.NewEntry(key, value).WithTTL(1 * time.Second)
```

Comment:
You can have e.WithTTL(1*time.Second) instead of creating NewEntry(). The rest looks good.

Reply:
Thanks. Done